adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/OpenApiDocs.kt

```diff
@@ -2,7 +2,8 @@ package net.adoptopenjdk.api.v3
 
 object OpenApiDocs {
 
-    const val RELASE_NAME = """
+    const val RELASE_NAME =
```

Comment:
This seems like an odd klint rule - do we really want this?

Reply:
Maybe not, and actually that reminds me, I noticed a number of rules being disabled in-line in the code. We could probably now just pull them, with any others we don't want into a file where we can control the rules?

For this PR, we could always just move the linting to Maven but stay on the same version and therefore the same default ruleset?

Reply:
For this rule, I actually think the alignment of the opening and closing `"""` is useful.

Reply:
I think to check with @johnoliver  on the disabled rules - I'd prefer to see them disabled in klint config as opposed to in code, but would be good to check.

Reply:
It seems that ktlint is very opinionated and I don't think there's actually support for a config file with custom rule-sets - see https://ktlint.github.io/#faq-error-suppression

This is probably why @johnoliver has suppressed them in-line where needed.

My thoughts are if we are using `ktlint` then maybe we just adopt its style completely (which includes [no wildcard imports](https://ktlint.github.io/#rule-import) btw), except for when there is a really strong reason not to. In these exceptional cases, which should be afew and far between, an in-line suppression may be ok.

If we do want more flexibility and custom rule-sets then I can recommend [detekt](https://github.com/detekt/detekt).